HomeWork/src/main/java/io/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,105 @@
+package io;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    String fileName = "NIO.txt";
+    private LoggingLevel loggingLevel;
+    private LoggingLevel[] arrayLevels;
+    protected int maxSizeFile = 1024;
+    private String formatFile = " [ПОТОЧНИЙ_ЧАС][LEVEL] ";
+
+    protected File logFile = new File(fileName);
+
+    public FileLoggerConfiguration() {
+    }
+
+    public FileLoggerConfiguration(String fileName, LoggingLevel loggingLevel, int maxSizeFile, String formatFile) {
+        if (LoggingLevel.DEBUG.equals(loggingLevel)) {
+            arrayLevels = new LoggingLevel[2];
+            arrayLevels[0] = LoggingLevel.INFO;
+            arrayLevels[1] = LoggingLevel.DEBUG;
+        } else {
+            arrayLevels = new LoggingLevel[1];
+            arrayLevels[0] = LoggingLevel.INFO;
+        }
+        this.fileName = fileName;
+        this.loggingLevel = loggingLevel;
+        this.maxSizeFile = maxSizeFile;
+        this.formatFile = formatFile;
+    }
+
+    public boolean isDebugEnabled() {
+        return contains(LoggingLevel.DEBUG);
+    }
+
+    public boolean isInfoEnabled() {
+        return contains(LoggingLevel.INFO);
```

Comment:
`INFO` should be enabled for the cases, when LoggingLevel = `INFO` and LoggingLevel = `DEBUG`